### ISSUE TITLE
Native find support

### ISSUE THREAD

**Issue by lengstrom** [nwjs/nw.js#2062]
Does NW support any sort of find system? 

Example: In Google Chrome, when you press control - f, you get a find window that shows a series of highlighted words on the site that match your pattern. Does node-webkit have anything like this?
##

**Comment by eriegz** [comment]
There's nothing in the above release 0.13 documentation that says anything about native find support being added. Was this feature actually added? If so, in which release? Where's the documentation?

**Comment by PriteshGajjar008** [comment]
Can someone confirm if this works? I tried but didn't work for me. I don't see any UI options on clicking Ctrl+F.

**Comment by PriteshGajjar008** [comment]
Its not working with latest version 0.21.3. Please advice.

**Comment by PriteshGajjar008** [comment]
Any updates on this?

**Comment by PriteshGajjar008** [comment]
I see this is reopened. @Christywl   Can you provide your view on this issue please?

**Comment by Christywl** [comment]
Yes, I tried on nwjs-sdk-v0.21.3, it doesn't work. Users can't use `CTRL-F` to search for a word.

**Comment by PriteshGajjar008** [comment]
Okay. Let us know when you guys are planning to work on this.

**Comment by rogerwang** [comment]
Thanks for your patience. We plan to fix it soon by providing an API and you can bind it to the hotkey. It shouldn't be a default behavior.

**Comment by elpeleq42** [comment]
@rogerwang So, any news on this? Couldn't find on documentation/google

**Comment by franck34** [comment]
+1

**Comment by rogerwang** [comment]
This is depending on #5875 , which has some progress recently. This feature should be easy to implement after that one is done.

**Comment by colorao** [comment]
+1

**Comment by truesolarflame** [comment]
+1

**Comment by TheJaredWilcurt** [comment]
I created an easy to use tool as a temporary stand in.

* * *

![find-in-nw](https://user-images.githubusercontent.com/4629794/67647035-29082e00-f907-11e9-9643-b81a23faa4f3.gif)

* * *

# Usage

1. `npm install --save findinnw`
1. In your HTML file add:
    ```html
    <script src="node_modules/findinnw/dist/find-in-nw.js"></script>
    <script>
      findInNw.initialize();
    </script>
    ```
1. Use `CTRL+F` and `ESC` to show/hide the search box

* * *

# API

* https://github.com/nwutils/find-in-nw

**Comment by prominentdetail** [comment]
any update on this?

**Comment by rogerwang** [comment]
I'll start adding browser features (including this) after #7230 is resolved.

**Comment by calumsult** [comment]
Is there an opportunity to fund this work?

**Comment by rogerwang** [comment]
@calumsult you could put a bounty on this feature via BountySource or IssueHunt.

**Comment by yushghimire** [comment]
any updates on this?

**Comment by DavidAccola** [comment]
Checking in on this...
